recipes/pymelts/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "pymelts" %}
+{% set version = "0.1.1" %}
+{% set checksum = "c31da5b4c2ed8cc9fdfc891b937431bebde05f2cc80ddf0dfe8f61aaacb910f5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://repository.ofm-research.org/packages/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 1
```

Comment:
This needs some sort of installation step. Please see the example included in this repo for an idea of how to do this.

xref: https://github.com/conda-forge/staged-recipes/blob/a172ea2b0031eb31d79476efa7fe731e566bc204/recipes/example/meta.yaml#L26